iOS Client/Models/MBPodcast.swift

```diff
@@ -18,3 +19,14 @@ struct MBPodcast: Codable, Detailable {
     let pubDate: String?
     let title: String?
 }
+
+struct DisplayablePodcast: Detailable {
+    let author: String?
+    let duration: String?
+    let guid: String?
+    let image: UIImage
+    let keywords: String?
+    let summary: String?
+    let pubDate: Date
+    let title: String?
+}
```

Comment:
hmmmm, we need a DRYer way to do this . . . Maybe just make the image's type `Data` instead of UIImage (since `Data` is codable) and keep this one struct

Reply:
dude, these are like identical models, anyway we can get away with just having one? 

Looks like pubDate is a date on DisplayablePodcast and that's the only difference, could just be a lazy property on the first struct that gets derived from the string date

Reply:
Hmm, I'm actually not sure. Maybe if we rename MBPodcast to be called `PodcastDTO` (data transfer object) and rename DisplayablePodcast to be `Podcast`, I'll be okay with this